src/oscar/apps/dashboard/app.py

```diff
@@ -54,6 +52,9 @@ def get_urls(self):
             }, name='logout'),
 
         ]
+        if is_promotions_enabled():
+            promotions_app = get_class('dashboard.promotions.app', 'application')
```

Comment:
How about `self.promotions_app or get_class(..)` for backwards compatibility with possible overrides?

Reply:
I'd probably check this case with the test.